experimental/python/databricks/bundles/compute/_models/environment.py

```diff
@@ -20,27 +16,16 @@ class Environment:
     In this minimal environment spec, only pip dependencies are supported.
     """
 
-    client: VariableOr[str]
-    """
-    Client version used by the environment
-    The client is the user-facing environment of the runtime.
-    Each client comes with a specific set of pre-installed libraries.
-    The version is a string, consisting of the major client version.
-    """
-
```

Comment:
@kanterov FYI, this property is removed because the underlying field is deprecated.

But it will break user code if used. They should probably still be emitted to keep things portable.